lib/experimental/fonts/font-library/font-library.php

```diff
@@ -57,6 +57,24 @@ function wp_register_font_collection( $config ) {
 	}
 }
 
+if ( ! function_exists( 'wp_unregister_font_collection' ) ) {
+	/**
+	 * Unregister Font Collection from the Font Library.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $collection_id The font collection's id.
+	 */
+	function wp_unregister_font_collection( $collection_id ) {
+		add_action(
+			'init',
```

Comment:
It's because we would need to be sure that all the font collections registered to call `wp_register_font_collection()` were already registered so we can unregister them using `wp_unregister_font_collection()`. Is there a better way to be sure that all the font collections are already registered?

Reply:
In the latest commit, I removed the reliance on the hooks by keeping track of the unregistered font collection using an array of collection IDs.

Reply:
Based on the @spacedmonkey comments, I changed the approach following [this](https://github.com/WordPress/wordpress-develop/blob/ce8aed4698117fd0f8c362825e62d01586c8799d/src/wp-includes/class-wp-block-patterns-registry.php#L130-L153) example.